src/codeu/chat/DerbyStore.java

```diff
@@ -35,23 +35,24 @@
 	private final String messageTableName = "message";
 	private final String chatParticipantsTableName = "chatParticipants";
 	
-	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getConversationInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getUsersInvolved = "SELCT * FROM " + chatParticipantsTableName + " WHERE id = ?" ;
+	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE userid = ?";
```

Comment:
nit: This is okay for our purposes, but with more complicated tables you probably want to select specific columns instead of selecting *.